flytekit/tools/translator.py

```diff
@@ -542,8 +542,8 @@ def get_reference_spec(
 
 def get_serializable(
     entity_mapping: OrderedDict,
-    settings: SerializationSettings,
     entity: FlyteLocalEntity,
+    settings: Optional[SerializationSettings] = SerializationSettings(ImageConfig()),
```

Comment:
Can we make this `None` instead of a dummy `SerializationSettings`?

Reply:
good catch, thanks! will update